python/lsst/sims/maf/metrics/moMetrics.py

```diff
@@ -543,7 +543,7 @@ def run(self, ssoObs, orb, Hval):
             # Count the number of observations per night (except last night)
             obsPerNight = (nIdx - np.roll(nIdx, 1))[1:]
             # Add the number of observations on the last night.
-            obsLastNight = np.array([len(visSort) - nIdx[-1]])
+            obsLastNight = np.array([len(nights) - nIdx[-1]])
```

Comment:
I changed this so it was consistent with the old metric. The length shouldn't ever be different, it was more of a cosmetic change so perhaps "incorrect" wasn't the right term to use in the commit message.